### PR TITLE
Fixed Deprecated integrations playbooks and scripts

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28226

## Description
fix all deprecated integrations playbooks and scripts
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Review by guykeller — APPROVED** [review]

**Comment by guykeller** [comment]
@ronykoz Waiting for your call here, looks good to me

File: Packs/Base/ReleaseNotes/1_1_14.md
Comment (ronykoz) [inline Packs/Base/ReleaseNotes/1_1_14.md:7]: i think that it should be 
```suggestion
#### Scripts
##### SaneDocReports
- Maintenance and stability enhancements
##### DBotMLFetchData
- Maintenance and stability enhancements
##### DBotSuggestClassifierMapping
- Maintenance and stability enhancements
##### SanePdfReports
- Maintenance and stability enhancements
```

File: Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage.yml
Comment (ronykoz) [inline Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage.yml:6]:
```suggestion
display: Cofense Triage (Deprecated)
```

File: Packs/CofenseTriage/ReleaseNotes/1_1_4.md
Comment (ronykoz) [inline Packs/CofenseTriage/ReleaseNotes/1_1_4.md:3]: here as well

**Review by ronykoz — APPROVED** [review]
please merge this only after you merge the SDK validation + an SDK version is released

**Review by ronykoz — APPROVED** [review]

**Comment by ronykoz** [comment]
@ChanochShayner please resolve conflicts

**Comment by ronykoz** [comment]
resolve conflicts please

**Review by ronykoz — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
@ronykoz @ChanochShayner Who decided we need to deprecate this? I am getting questions from some of the vendors here

**Pull request merged**
